libs/core/src/lib/datetime/fd-date.ts

```diff
@@ -59,6 +65,18 @@ export class FdDate {
         );
     }
 
+    static isValidByParams(params: {year: number, month?: number, day?: number, hour?: number, minute?: number, second?: number}): boolean {
```

Comment:
Not sure if it worths exposing to the public API.
Let's hide it from the public API

Reply:
yes, should be hiddem